src/org/mozilla/javascript/FunctionObject.java

```diff
@@ -91,7 +91,7 @@ public FunctionObject(String name, Member methodOrConstructor,
         }
         String methodName = member.getName();
         this.functionName = name;
-        Class<?>[] types = member.argTypes();
```

Comment:
Not on this line but...

If we're doing a "cleanup" PR, can we also clean up "import *" stuff? I see that this file and others have it. I'd love to update the checkstyle rule to disallow it (as we do in most Java codebases I've worked on) but there is too much of it in Rhino!

Reply:
As you wish master ;-)